Signal/src/views/NotificationView.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+import Foundation
+
+class NotificationView: UIView {
```

Comment:
I'm wondering if there's a better name for this - `NotificationView` is probably fine but makes me think of NSNotifications, local notifications, remote push notifications.  Maybe: `WarningView`, `NagView`, etc., etc.